### PR TITLE
FOGL-5758 Fix order for plugin configuration

### PR DESCRIPTION
**Pull request by MarkRiddoch**
Signed-off-by: Mark Riddoch <mark@dianomic.com>

### CODE REVIEW

File: plugin.cpp [hunk 1]
```diff
@@ -38,64 +38,64 @@ static const char *default_config = QUOTE({
...
 			"type": "boolean",
 			"default": "False",
-			"order": "14",
+			"order": "8",
 			"displayName": "Verify SSL"
 			},
 		"applyFilter": {
         		"description": "Whether to apply filter before processing the data",
 			"type": "boolean",
 			"default": "False",
```
Comment (YashTatkondawar) [inline plugin.cpp:90]: Shouldn't it be "false" instead of "False"?

File: plugin.cpp [hunk 2]
```diff
@@ -38,64 +38,64 @@ static const char *default_config = QUOTE({
...
 			"type": "integer",
 			"default": "10",
-			"order": "13",
+			"order": "7",
 			"displayName": "Http Timeout (in seconds)"
 			},
 		"verifySSL": {
         		"description": "Verify SSL certificate",
 			"type": "boolean",
 			"default": "False",
```
Comment (YashTatkondawar) [inline plugin.cpp:83]: Shouldn't it be "false" instead of "False"?

File: plugin.cpp [hunk 3]
```diff
@@ -38,64 +38,64 @@ static const char *default_config = QUOTE({
...
 			},
 		"applyFilter": {
         		"description": "Whether to apply filter before processing the data",
 			"type": "boolean",
 			"default": "False",
-			"order": "15",
+			"order": "9",
 			"displayName": "Apply Filter"
 			},
 		"filterRule": {
```
Comment (YashTatkondawar) [inline plugin.cpp:94]: We may also add validity expression such as `"validity": "applyFilter == \"true\""`

**Review by YashTatkondawar — APPROVED** [review]
@MarkRiddoch I have added few comments. Please let me know if those can be fixed in this branch or shall I file a new JIRA?

File: plugin.cpp (hunk 1 above)
Comment (MarkRiddoch) [reply plugin.cpp:90]: We do a case independent check, so it should be fine.

File: plugin.cpp (hunk 3 above)
Comment (MarkRiddoch) [reply plugin.cpp:94]: Yes, this is really obsolete now we have filters in the north. The intention is to remove this for 2.0

**Pull request merged**